### PR TITLE
sysinternals@2022.11.10: Fix hash

### LINKED ISSUE #84: sysinternals@2022.11.10: hash check failed
Checking hash of SysinternalsSuite.zip ... ERROR Hash check failed!
App:         business/sysinternals
URL:         https://download.sysinternals.com/files/SysinternalsSuite.zip
First bytes: 50 4B 03 04 14 00 00 00
Expected:    291e6741d6e5b9895a007c398b849667673b1c8f8cd0e3b8571fbb015008cf53
Actual:      76e374b9844fa1e6491cc3e70d73f9ce885253258b86d277d84e40452ba8b8f5

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #84

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### sysinternals

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
